### PR TITLE
Issue 7

### PR DESCRIPTION
**Pull request by dcrystalj**

### CODE REVIEW

**Comment by mitar** [comment]
Is this finalized code?

**Comment by dcrystalj** [comment]
I'm not sure. It does what it's written in issue#7- after user click allow you can get information about user. But I'm not really sure how to access that data later. Also I still have no idea if it's correct to put script code directly in .html file like I did here in django.

**Comment by mitar** [comment]
No. Move code to external files and made Facebook app id configurable through `settings.py`.

File: piplmesh/frontend/templates/header.html [hunk 1]
```diff
@@ -13,4 +13,24 @@
                 </div>
             </form>
         </div>
-    </div>
+		<div id="fb-root"></div>
```
Comment (mitar) [inline piplmesh/frontend/templates/header.html]: Are we indenting with four or two spaces? Please spell out whole names, be verbose!

**Comment by mitar** [comment]
Closing in favor of other pull requests.

**Pull request closed without merging**